### PR TITLE
upkeep/40: PHP8 compat workflow

### LINKED ISSUE #40: Add PHP8 Compatibility testing workflow
### Is your enhancement related to a problem? Please describe.

- [x] Add workflow for PHP8 Compatibility testing
- [x] Perform a round of manual testing to verify the plugin doesn't break on PHP8

### Designs

_No response_

### Describe alternatives you've considered

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Sidsector9**
### Description of the Change

-  Added PHP8 Compatibility workflow
- Bumped minimum WP version to 5.8
- Bumped minimum PHP version to 5.6

### Checklist:

- [x] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [x] All new and existing tests passed.

### Changelog Entry
upkeep: Added PHP8 compatibility workflow

### Credits

Props @Sidsector9

### CODE REVIEW

File: retro-webamp-block.php
Comment (faisal-alvi) [inline retro-webamp-block.php:7]: The version should also be bumped in readme files.

https://github.com/10up/retro-winamp-block/blob/2d613feef8a03829524c6bef7f72b89f2334d970/readme.txt#L4

https://github.com/10up/retro-winamp-block/blob/2d613feef8a03829524c6bef7f72b89f2334d970/README.md?plain=1#L18

**Review by faisal-alvi — DISMISSED** [review]
@Sidsector9 LGTM - just requires a version bump before the merge. Approving now, as the change is minor.

**Review by faisal-alvi — APPROVED** [review]

File: retro-webamp-block.php
Comment (jeffpaul) [inline retro-webamp-block.php:8]: Should this get changed to 5.6?

**Pull request merged**
